e2e/utils/strategy/template.ts

```diff
@@ -61,17 +65,23 @@ const fromLimitOrder = (order: LimitOrder): RangeOrder => ({
   budget: order.budget,
 });
 export const createDebugStrategy = {
-  limitBuy: (pair: TokenPair, buy: LimitOrder): CreateStrategyTemplate => ({
+  recurringLimitBuy: (
```

Comment:
same comment as above. I would leave this file as it for now, we can improve it later when unifying the naming